src/graph.py

```diff
@@ -246,7 +246,10 @@ def _prepare_prizes(self, prizes_dataframe):
         logger.info(prizes_dataframe[prizes_dataframe.index == -1]['name'].tolist())
         prizes_dataframe.drop(-1, inplace=True, errors='ignore')
 
-        self.node_attributes = prizes_dataframe.set_index('name').rename_axis(None)
+        # Node attributes dataframe for all proteins in self.nodes
+        self.node_attributes = prizes_dataframe.set_index('name').rename_axis(None).reindex(self.nodes)
+        self.node_attributes["prize"].fillna(0, inplace=True)
+        self.node_attributes["type"].fillna("steiner", inplace=True)
```

Comment:
What happens if there isn't a `prize` or `type` column in the dataframe?

Reply:
`prize` is required to be a column. `type` should probably be required as well, but if it's not specified, all prizes are set to "terminal".

31ed9921146fc12ca9c71af3257cece4adff32b2